run.sh

```diff
@@ -15,7 +15,8 @@ fi
 
 exec 2>&1
 
-java -jar /opt/exhibitor/exhibitor.jar \
+java -javaagent:/agents/appdynamics-jvm/javaagent.jar -Dappdynamics.agent.uniqueHostId=$(cat /agents/appdynamics-jvm/uniqueHostId) \
```

Comment:
I made AppDynamics optional, i.e. if agent is not available or Senza parameter is not set.